content/articles/compose-for-desktop/index.md

```diff
@@ -0,0 +1,347 @@
+Desktop applications are applications that run on desktop machines. They are being used almost everywhere for example a school management system. Before Compose Desktop came into existence, the Java SWING library was used to develop these desktop applications. With the new technology of Compose Desktop coming into existence we can have a choice of which technology to use when we want to develop a desktop app.
+
+Compose Desktop allows us to develop desktop apps with lovely user interfaces like the ones in Android apps. Compose for Desktop can be used to create desktop applications for Windows, macOS, or Linux platforms. Kotlin multiplatform also allows for building Compose Web Application and Compose Multiplatform Application.
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+- [Step 1 - Creating Project](#step-1---creating-project)
+- [Step 2 -  Understanding Compose Desktop Application Project Structure](#step-2---understanding-compose-desktop-application-project-structure)
+- [Step 3 - User Login Class](#step-3---user-login-class)
+- [Explanation](#explanation)
+- [Step 4 - User Registration Class](#step-4---user-registration-class)
+- [Step 5 - Main Class](#step-5---main-class)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+To read and understand this tutorial, you must have:
+- [IntelliJ](https://www.jetbrains.com/idea/download/#section=windows) IDEA installed
+- [JDK version 11](https://www.oracle.com/java/technologies/downloads/) or later installed.
+- Basic knowledge in [Jetpack Compose](https://developer.android.com/jetpack/compose)
+
+### Getting Started
+In this tutorial, we will learn to use Compose for Desktop by building simple user authentication screens. We will therefore learn how to create some basic views like `TextField` and `Button` in a Compose Desktop app. This tutorial will act as a beginner guide to the introduction to Compose for Desktop. 
+
+IntelliJ IDEA has built-in Kotlin plugins and Compose Desktop libraries. 
+
+### Step 1 - Creating Project
+To create a Compose Desktop Application, open your IntelliJ IDEA, select `New Project` then select `Kotlin` programming language on the left pane, give your project a name. Remember to select `Compose Desktop Application` in the `Project Template` just below the project location path the click next.
```

Comment:
, and give the project a name.
(Remember to select `Compose Desktop Application` in the `Project Template` just below the project location path the click next.) I'm not sure if the sentence has a perfect meaning, consider rephrasing it.